doc/reference/netplan-yaml.md

```diff
@@ -2192,4 +2192,4 @@ consumer of that back end. Currently, this is only used with `NetworkManager`.
 
     > Can be used as a fallback mechanism to missing key-file settings.
 
-<!--- vim: ft=markdown -->
+<!--- vim: ft=markdown -->
```

Comment:
nitpick (non-blocking): This change seems to be unrelated and we should consider dropping it.

```diff
-<!--- vim: ft=markdown -->
+<!--- vim: ft=markdown -->
\ No newline at end of file
```

Reply:
Hmmm, I didn't add this myself (Tehnically, I did, to resolve the conflict). It's part of the current netplan-yaml file on the main branch.